mirrord/config/src/experimental.rs

```diff
@@ -40,7 +40,7 @@ pub struct ExperimentalConfig {
     /// ### _experimental_ hide_ipv6_interfaces {#experimental-hide_ipv6_interfaces}
```

Comment:
should it still be experimental if it's enabled by default?

Reply:
yes, so users can turn it off if it affects them badly until we roll a fix.